Formula/aap.rb

```diff
@@ -1,5 +1,5 @@
 class Aap < Formula
-  desc "make-like tool to download, build, and install software"
+  desc "Tool similar to make to download, build, and install software"
```

Comment:
I think this could be `Make-like tool`. Generally any case where it gets reordered to avoid a name being uppercased: let me know as it should probably be added to the exception list.